modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/objectDefinitionUtil.ts

```diff
@@ -131,11 +131,26 @@ export async function deleteRelationship(id: number) {
 				'relationship-was-deleted-successfully'
 			),
 		});
+
+		setTimeout(() => window.location.reload(), 1500);
```

Comment:
Hey @larissacribeiro, sorry, I've noticed one more thing, in Model Builder this will cause the entire screen to be reloaded after deletion, we don't need this as the React Context API handles the state change. So we need to pass a new optional parameter here to this `deleteRelationship` function, and if true we run `setTimeout`. In this case, the only place we will pass true is in the `Relationship.tsx` line `176` 

```tsx

export async function deleteRelationship(
	id: number,
	reloadAfterDeletion?: boolean
) {
...
}
```

```suggestion
		if(reloadAfterDeletion) {
			setTimeout(() => window.location.reload(), 1500);
		}
```

Reply:
Hey @danvb112, got it, np! I'v tested and updated it right now 😄